exercises/connect/example.tt

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'connect'
+
+# Test data version:
+# <%= sha1 %>
```

Comment:
Can you move the sha1 id to line 6 so it's all on one line.  
And add an indicator that it's a commit id.
`# Test data version commit id: 15caeda`